urdf_mesh_converter/CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(urdf_mesh_converter)
+
+Find_package(
+  catkin REQUIRED
+  catkin_virtualenv
+)
+
+if (${catkin_virtualenv_VERSION} VERSION_LESS "0.5.1")
+  message(STATUS "${PROJECT_NAME} requires catkin_virtualenv >= 0.5.1")
+  return()
+endif()
+
+
+catkin_python_setup()
+
+catkin_package(
+  CATKIN_DEPENDS
+)
+
+if($ENV{ROS_DISTRO} STREQUAL "noetic")
+  catkin_generate_virtualenv(
+    INPUT_REQUIREMENTS requirements.in
+    PYTHON_INTERPRETER python3
+  )
+else()
+  message(FATAL_ERROR "This package uses trimesh for model conversion. Trimesh has discontinued support for Python 2 quite some time ago. Please use Python 3 or later.")
```

Comment:
How about using `message(WARNING ...` and `return()` to keep installability of whole `jsk_model_tools` into python2 systems?

Reply:
Thanks. Fixed.
https://github.com/jsk-ros-pkg/jsk_model_tools/pull/255/commits/b64b866662abb42d2622789f247704933559c595